src/basic/socket-util.c

```diff
@@ -900,6 +900,29 @@ bool ifname_valid(const char *p) {
         return true;
 }
 
+bool iflabel_valid(const char *p) {
+
+        if (isempty(p))
+                return false;
+
+        if (strlen(p) >= IFNAMSIZ)
+                return false;
+
+        if (dot_or_dot_dot(p))
+                return false;
+
+        while (*p) {
+                if ((unsigned char) *p >= 127U)
+                        return false;
+
+                if ((unsigned char) *p <= 32U)
```

Comment:
well, you'd have to enclose the label with "" if you include a space:

```
sudo  ip address add 192.168.99.37/24 brd + dev wlp3s0 label "wlp3s0 11"
```

And if I run that it does accept it